utils/math_functions.py

```diff
@@ -245,10 +245,10 @@ def stan_inverse(self,y,x):
 		return atan2(y,x)+3.14159265
 
 
-def getPointBehindTheBall(point ,theta):
-  x = point.x -(3.5 * BOT_RADIUS) *(math.cos(theta))
-  y = point.y -(3.5 * BOT_RADIUS) *(math.sin(theta))
-  return Vector2D(int(x), int(y))
+def getPointBehindTheBall(point, theta):
+	x = point.x - (1*BOT_RADIUS) * (math.cos(theta))
+	y = point.y - (1*BOT_RADIUS) * (math.sin(theta))
+	return Vector2D(int(x), int(y))
```

Comment:
Remove duplicate code

Reply:
Yeah, a merge with fsm branch to get this branch updated will fix this.